pipert/core/component.py

```diff
@@ -1,6 +1,9 @@
 import threading
 from prometheus_client import start_http_server
 from torch.multiprocessing import Event, Process
+
+from pipert.contrib.metrics_collectors.prometheus_collector import PrometheusCollector
+from pipert.contrib.metrics_collectors.splunk_collector import SplunkCollector
```

Comment:
The Splunk installation is optional. I think that this import will raise error if splunk is not installed.

Reply:
@AmitKarin you need to look how the the pipe images and docker compose file working now and check this when splunk is not installed.